hotel/hotel/src/main/java/com/hotel/hotel/Dto/RoomDto.java

```diff
@@ -0,0 +1,12 @@
+package com.hotel.hotel.Dto;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+
+@Data
+public class RoomDto {
+    private int idRoom;
+    private int numberOfBeds;
+    private int price;
```

Comment:
Usually the prices tend to be a decimal number instead of an integer, I recommend using BigDecimal type